src/System.IO.FileSystem/tests/WinRT_BrokeredFunctions.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+using Windows.Storage;
+
+namespace System.IO.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Yeah, that's the issue with these tests. When I first added some tests I thought it would be nice to override our FileCleanupTestBase temporary test base location with a brokered location to get full test coverage to brokered locations via inheritance, but decided it would be too dangerous and too flaky.

It does make me a bit nervous to have so many coverage gaps, still.